examples/pytorch/MNIST/mnist_autolog_example.py

```diff
@@ -16,7 +16,7 @@
 from pytorch_lightning.callbacks.early_stopping import EarlyStopping
 from pytorch_lightning.callbacks import ModelCheckpoint
 from pytorch_lightning.callbacks import LearningRateMonitor
-from pytorch_lightning.metrics.functional import accuracy
+from torchmetrics.functional import accuracy
```

Comment:
Can we revert this change since it's unrelated to this PR?

Yesterday I filed https://github.com/mlflow/mlflow/pull/4976 and fixed the issue in the example check.

Reply:
If #4976 didn't actually solve the issue and we need another fix, we can create a separate PR.

Reply:
@liangz1 On a second thought, can we revert the changes #4976 made, and apply your fix in a separate PR?

Reply:
Sure, I created https://github.com/mlflow/mlflow/pull/4989 for the fix and revert.